gradle/libs.versions.toml

```diff
@@ -6,21 +6,21 @@ compileSdk = "34"
 
 # Dependencies
 kotlin = "2.0.10"
-android-gradle-plugin = "8.2.2"
+android-gradle-plugin = "8.7.0"
```

Comment:
I thought I read something about not being able to use the latest gradle plugin because of KMP but I can't find it anymore, and it seems to run fine.

Reply:
Found it
https://kotlinlang.org/docs/multiplatform-compatibility-guide.html#version-compatibility

Reply:
Added a comment